src/client/client.js

```diff
@@ -182,6 +182,17 @@ export default class Client {
         return data;
     }
 
+    logout = async () => {
+        const response = await this.doFetch(
```

Comment:
Maybe use `this.doFetchWithResponse` here and check `response.ok` instead of `user_id` to make sure the call was successful? That way it'll be clearer what it's trying to do